ceph_cfg/presenter.py

```diff
@@ -349,3 +359,21 @@ def ceph_version(self):
 
     def cephfs_list(self):
         return self.model.cephfs_list
+
+
+    def df_osd(self, osd_number):
+        if self.model.cluster_df is None:
+            msg = "Programming error : self.model.cluster_df is None"
+            log.error(msg)
+            raise(msg)
+        osd_details = self.model.cluster_df.get("osd")
+        if osd_details is None:
+            msg = "Programming error : self.model.cluster_df[osd] is None"
+            log.error(msg)
+            raise(msg)
+        details = osd_details.get(osd_number)
```

Comment:
Yes it does stats for all OSD's.

I am not sure how expensive it it. I had not really thought about cost here as I wanted to focus on functionality.

It might be better to use another call for osd fullness in osd removal. I can list PG by OSD for a less costly operation.

For admin though I see utility in providing a call to do this for inspection.